pf/tfgen/main.go

```diff
@@ -121,181 +106,3 @@ func MainWithMuxer(provider string, infos ...tfbridge.Muxed) {
 		return nil
 	})
 }
-
-func mergeDisjoint[T any](dst *map[string]T, src map[string]T) {
-	contract.Assertf(dst != nil, "Cannot assign to a nil destination")
-	if src == nil {
-		return
-	}
-	if *dst == nil {
-		*dst = map[string]T{}
-	}
-	for k, v := range src {
-		_, ok := (*dst)[k]
-		contract.Assertf(!ok, "Map was not disjoint on key %s", k)
-		(*dst)[k] = v
-	}
-}
-
-// Mux multiple infos into a unified representation.
-//
-// This function exposes implementation details for testing. It should not be used outside
-// of pulumi-terraform-bridge.  This is an experimental API.
-func UnstableMuxProviderInfo(
```

Comment:
Thinking out loud: do you want to keep this around somewhere for the generalized provider muxing case - we don't have a use case lined up now but maybe in the future? If we're mixing two providers P1 and P2 without knowing much about them (not assuming they're bridged, say). This would require mixing their schemas at least. So perhaps the code to merge ProviderInfo is not needed there, and all this code is about that, not merging the schemas.

Reply:
This is only about merging `ProviderInfo`, which is bridge specific. Unless we envision a scenario where we are merging multiple bridged providers but can't do this by merging `shim.Provider` on a single `ProviderInfo` I don't think this will be used.

Schema mixing code in in `x/muxer`.